tools/travis/localua.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+# Downloads and installs a self-contained Lua and LuaRocks.
+# Supports Linux, macOS and MSYS2.
+# Copyright (c) 2015-2019 Pierre Chapuis, MIT Licensed.
+# Latest stable version available at: https://loadk.com/localua.sh
+# Maintained at: https://github.com/oploadk/localua
+
+DEFAULT_LUA_V="5.3.5"
```

Comment:
We're actually still using 5.1.4 but 5.3 is coming (#2783).

Reply:
This Lua version is only to run `luarocks`+ `luacheck` and in `luacheck_config` [here](https://github.com/nodemcu/nodemcu-firmware/pull/2790/files#diff-e9472bcdeadb819510dd8e9e8ae1869cR883) code to check is set to support Lua 5.1 (change for 5.3 will only require change `51` to `53`). I used 5.3 version because I'm not 100% sure if `luacheck` will work properly on older (according to docs it should) but that setting only matters for Travis. AFAIK most of linux environments allow to install multiple Lua versions but defaults to 5.3.